### PR TITLE
chore(fence-public): Add fence-config-public to PROD_NCT

### PR DESCRIPTION
**Pull request by atharvar28**
Adding `fence-config-public.yaml` to PROD NCT

### CODE REVIEW

**Review by paulineribeyre — APPROVED** [review]

File: accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml [hunk 1]
```diff
@@ -182,4 +182,19 @@ STORAGE_CREDENTIALS: []
 #       length restrictions on group names.
 GOOGLE_GROUP_PREFIX: 'gen3'
 
+S3_BUCKETS:
```
Comment (paulineribeyre) [inline accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml:185]: fix `ACCESS_TOKEN_EXPIRES_IN`, `REFRESH_TOKEN_EXPIRES_IN` and `S3_BUCKETS` in the QA config file too

File: accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml [hunk 2]
```diff
@@ -90,10 +90,10 @@ OAUTH2_TOKEN_EXPIRES_IN:
   "implicit": 1200
 
 # The number of seconds after an access token is issued until it expires.
-ACCESS_TOKEN_EXPIRES_IN: 1200
+ACCESS_TOKEN_EXPIRES_IN: 12000
 
 # The number of seconds after a refresh token is issued until it expires.
-REFRESH_TOKEN_EXPIRES_IN: 2592000
+REFRESH_TOKEN_EXPIRES_IN: 25920000
```
Comment (paulineribeyre) [inline accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml]: the default values in the fence default config are
```# The number of seconds after an access token is issued until it expires.
ACCESS_TOKEN_EXPIRES_IN: 1200

# The number of seconds after a refresh token is issued until it expires.
REFRESH_TOKEN_EXPIRES_IN: 2592000
```
why was this changed for NCT??

**Review by emalinowski — APPROVED** [review]

File: accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml (hunk 1 above)
Comment (atharvar28) [reply accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml:185]: 👍

File: accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml (hunk 2 above)
Comment (atharvar28) [reply accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml]: In the PROD VM it was set to these values. The change was to match the public config with fence-config in PROD VM

File: accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml (hunk 1 above)
Comment (atharvar28) [reply accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml:185]: Here is the PR to update the QA config file
https://github.com/uc-cdis/gitops-qa/pull/1442

File: accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml (hunk 2 above)
Comment (paulineribeyre) [reply accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml]: we should use the default values in QA and Prod 👍

**Review by emalinowski — APPROVED** [review]

**Review by paulineribeyre — APPROVED** [review]
